controllers/mod.js

```diff
@@ -176,15 +170,15 @@ exports.postClearModPlants = (req, res, next) => {
         else if (data.selection == 5) {
           p=plant5
         }
-        console.log(ip[x]);
+        // console.log(ip[x]);
```

Comment:
Probably be nice to just junk these unused console logs, easy to rewrite later and will clean up the file